nncf/torch/pruning/operations.py

```diff
@@ -281,6 +281,33 @@ def output_prune(cls, model: NNCFNetwork, node: NNCFNode, graph: NNCFGraph) -> N
 class PTLinearPruningOp(LinearPruningOp, PTPruner):
     subtypes = [PTLinearMetatype, PTMatMulMetatype]
 
+    @classmethod
+    def input_reorder(cls, model: NNCFNetwork, node: NNCFNode, graph: NNCFGraph):
+        input_masks = get_input_masks(node, graph)
+        reorder_indexes = input_masks[0]
+        if reorder_indexes is None:
+            return
+        reorder_indexes = reorder_indexes.tensor
+        reorder_indexes = reorder_indexes.int()
```

Comment:
Why is `int()` only needed for `input_reorder`?

Reply:
@vinnamkim Thanks! We have removed int() since it is not required here.